server/src/main/java/io/seata/server/ServerApplicationListener.java

```diff
@@ -94,6 +96,12 @@ public void onApplicationEvent(ApplicationEvent event) {
         }
         String servicePort = String.valueOf(Integer.parseInt(serverPort) + SERVICE_OFFSET_SPRING_BOOT);
         setTargetPort(environment, servicePort, true);
+
+        // Load by priority
+        System.setProperty("sessionMode",
```

Comment:
![image](https://user-images.githubusercontent.com/24909409/146357896-ea102fbf-6703-463f-b961-abc54963482f.png)
if I config  `service-port` , this line will not execute
```
 server:
    service-port: 8091
```